### PR TITLE
Bunch of resource stuff

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/net/minecraft/client/resource/remote/Downloader.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_qcnikyod net/minecraft/client/resource/remote/Downloader
+	METHOD m_ijvtzctw queueDownload (Ljava/util/Map;Ljava/util/function/Consumer;)V
+		ARG 1 downloads
+		ARG 2 callback
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/resource/remote/Downloader.mapping]:
```suggestion
		ARG 2 onComplete
```
more specific

File: mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		ARG 2 url
+	METHOD m_jdpengrg createDownloadedFuture (Ljava/util/UUID;)Ljava/util/concurrent/CompletableFuture;
+	METHOD m_kneyvfit success ()V
+	METHOD m_knzlaroh addLocalPack (Ljava/util/UUID;Ljava/nio/file/Path;)V
+		ARG 2 path
+	METHOD m_lvunqimi startReload (Lnet/minecraft/unmapped/C_mjfziaze$C_waclobzb;)V
+		ARG 1 callback
+	METHOD m_oklkcsby (Ljava/util/function/Consumer;)V
+		ARG 0 consumer
+	METHOD m_rzyzxtfp removeAllPaacks ()V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping]:
```suggestion
	METHOD m_rzyzxtfp removeAllPacks ()V
```

File: mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		FIELD f_ewnjbkhb totalSize Ljava/util/OptionalLong;
+		FIELD f_fkcwysli downloads I
+		FIELD f_fxdywpla additionalInfo Lnet/minecraft/unmapped/C_rdaqiwdt;
+		FIELD f_tonxqzfr failures I
+		FIELD f_wzcqodju title Lnet/minecraft/unmapped/C_rdaqiwdt;
+		METHOD m_hcmuvepm updateToast (J)V
+			ARG 1 bytes
+		METHOD m_stgjfbkz showToast ()V
+	CLASS C_ooojwodg
+		FIELD f_bojtpdfw MAXA_SIZE I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping]:
```suggestion
		FIELD f_bojtpdfw MAX_SIZE I
```

File: mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping [hunk 4]
```diff
@@ -16,6 +16,9 @@ CLASS net/minecraft/unmapped/C_phqvxqun net/minecraft/server/integrated/Integrat
 	METHOD m_pemmxxxv askForBackup (Lnet/minecraft/unmapped/C_bpwagnit$C_unecclfw;ZLjava/lang/Runnable;Ljava/lang/Runnable;)V
 		ARG 2 legacyCustomized
 		ARG 3 callback
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping]:
```suggestion
		ARG 1 session
		ARG 2 legacyCustomized
		ARG 3 onProceeded
		ARG 4 onCancelled
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping (hunk 4 above)
Comment (OroArmor) [reply mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping]: Are you wanting to add these mappings?

File: mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/client/resource/remote/RemoteResourcePackProvider.mapping]: i'm not sure how this got through, it's not in either wordlist

File: mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping (hunk 4 above)
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping]: Yeah, I did a `callback` search after suggesting changing that other param, and ended up mapping this.

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]
looks great!

**Review by IotaBread — APPROVED** [review]

**Pull request merged**